Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
 import './Header.css';
+import moment from 'moment';
+
+const HeaderTitle = () => {
+    return (
+        <div className ='header-info'>
+            <h1>Lambda School</h1>
+            <p className='atlambda'>@LambdaSchool</p>
+            <p>&nbsp;&#8226; {moment().format("Do MMM")}</p>
```

Comment:
I notice that you're using the moment library. I hope it wasn't too much documentation to read through. Great job!